packages/material-ui-envinfo/package.json

```diff
@@ -15,9 +15,15 @@
   "bugs": {
     "url": "https://github.com/mui-org/material-ui/issues"
   },
+  "homepage": "https://github.com/mui-org/material-ui/tree/HEAD/packages/material-ui-envinfo",
   "repository": {
     "type": "git",
     "url": "https://github.com/mui-org/material-ui",
     "directory": "packages/material-ui-envinfo"
+  },
+  "files": [],
+  "scripts": {
+    "build": "npm pack && tar --transform 's,package/,build/,' -xzf material-ui-envinfo-*.tgz",
```

Comment:
`tar --transform` is not working on macOS: https://superuser.com/questions/318809/linux-os-x-tar-incompatibility-tarballs-created-on-os-x-give-errors-when-unt.

Reply:
I have resolved the issue in my env by installing a new dependency and overriding the default command.

Reply:
Can you install the GNU tar on your system?

Reply:
I have installed it under `gtar` with brew. However I can't replace tar, it's under an integrity protection system of macOS, I'm scared to override it.

Does it work with Windows?

Reply:
I even tested it under windows :sweat: : https://github.com/mui-org/material-ui/pull/23886#discussion_r538317380

But I didn't think about different tar implementations :grin: This was more as a quick unblock-suggestion not something that was meant to last. Long-term solution is in https://github.com/mui-org/material-ui/pull/24002